add-chain/cmd/check-genesis-config.go

```diff
@@ -0,0 +1,66 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/big"
+	"os"
+	"reflect"
+
+	"github.com/ethereum-optimism/superchain-registry/add-chain/flags"
+	"github.com/ethereum/go-ethereum/core"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"github.com/urfave/cli/v2"
+)
+
+var CheckGenesisConfigCmd = cli.Command{
+	Name:  "check-genesis-config",
+	Flags: []cli.Flag{flags.GenesisConfigFlag, flags.ChainIdFlag},
+	Usage: "Sanity check genesis config (genesis.json) is reproducible",
+	Action: func(ctx *cli.Context) error {
+		genesisPath := ctx.String(flags.GenesisConfigFlag.Name)
+		fmt.Printf("Attempting to read from %s\n", genesisPath)
+		file, err := os.ReadFile(genesisPath)
+		if err != nil {
+			return fmt.Errorf("failed to read from local genesis.json config file: %w", err)
+		}
+		var localGenesis *core.Genesis
+		if err = json.Unmarshal(file, &localGenesis); err != nil {
+			return fmt.Errorf("failed to unmarshal local genesis.json into core.Genesis struct: %w", err)
+		}
+
+		chainId := ctx.Uint64(flags.ChainIdFlag.Name)
+		gethGenesis, err := core.LoadOPStackGenesis(chainId)
+		if err != nil {
+			return fmt.Errorf("failed to load genesis via op-geth: %w", err)
+		}
+
+		opts := cmp.Options{
+			// ExcessBlobGas, BlobGasUsed: allow &0 == nil
```

Comment:
Why do we want to make an exception for this?

Reply:
This was a work-around that is no longer needed. Removed the exception: https://github.com/ethereum-optimism/superchain-registry/pull/354/commits/eaf83cece4b7612c39eeeb63fb1e1f3fd954f6f5